modules/version-update/pages/update-tool.adoc

```diff
@@ -0,0 +1,516 @@
+= Updating Bonita Runtime 
+:description: This page explains how to update Bonita Runtime version (single or multiple nodes).
+
+This page explains how to update Bonita Runtime version (single or multiple nodes).
+
+[NOTE]
+====
+
+.*Updating* means moving from one Bonita version to a newer one, to benefit from:
+* new features (in Main versions)
+* technology updates (in Main versions)
+* bug fixes (in Maintenance versions)
+ 
+====
+
+== How it works
+
+_Bonita Update Tool_ contains a set of scripts that execute changes on the data for making it compatible with a newer version of Bonita Runtime. 
+The tool is provided as a zip archive that contains an executable script to determine your current version and request the targeted Bonita version for the update.
+
+The update process is step-based, with each maintenance version being equal to a step. +
+The script manages the sequence of steps from your current version to the targeted version.
+After each main or maintenance version update, you have the option to pause or continue to the next step.
```

Comment:
This is not true, you cannot pause on each step

Reply:
removed. Thank for the heads up.